pkg/sql/util/util.go

```diff
@@ -44,6 +44,15 @@ func SplitTableAndColumn(name string) (string, string) {
 	return schema, xs[len(xs)-1]
 }
 
+func TableIsLoggingTable(dbName string, tableName string) bool {
```

Comment:
This is alslo wrong -- why did engine need to know these tables?   They are not special in any ways.